plasmapy/atomic/atomic.py

```diff
@@ -231,8 +233,13 @@ def isotope_symbol(argument, mass_numb=None):
         raise ValueError("Insufficient information to determine element and "
                          "mass number in isotope_symbol.")
 
+    if __is_neutron(argument) and (mass_numb is None or mass_numb == 1):
+        return 'n'
+    elif argument == 0 and mass_numb == 1:
+        return 'n'
```

Comment:
Oh, so you did add it! One of my previous comments may have just been invalidated, then!